src/float_parser.ts

```diff
@@ -0,0 +1,39 @@
+import BN from "bn.js";
+
+const buf = new ArrayBuffer(8);
+const view = new DataView(buf);
```

Comment:
I'd prefer to inline them to where they are used, since they are not actually constant and the extra allocation is likely not going to be the bottleneck.